src/strings.rs

```diff
@@ -252,7 +259,7 @@ pub fn normalize_label(i: &str, preserve_case: bool) -> String {
             }
         } else {
             last_was_whitespace = false;
-            if preserve_case {
+            if Case::Preserve == casing {
                 v.push(c);
             } else {
                 v.push_str(&c.to_lowercase().to_string());
```

Comment:
You could also consider something like this in future, I think it's probably considered more "idiomatic Rust":

```suggestion
            match casing {
                Case::Preserve => v.push(c),
                Case::DontPreserve => v.push_str(&c.to_lowercase().to_string()),
```

Reply:
Ah thanks. I went ahead and applied this to the original commit and force pushed.